css/style.css

```diff
@@ -395,19 +381,70 @@ h1,h2,h4 {
     flex-direction: column;
 }
 
-.content__reasons
-{   font-size: 40px;
+.content__title {
+    font-size: 40px;
     font-weight: 700;
     line-height: 55px;
-    
+    text-transform: uppercase;
+    margin: 145px 0 10px 0;
 }
 
-p {
+.content__text {
     font-size: 14px;
+    font-style: normal;
     font-weight: 500;
-    line-height: 20px; 
+    line-height: 20px;
     letter-spacing: 0.2px;
 }
 
+.grid-container {
+    display: grid;
+    align-content: space-around;
+    grid-template-columns: auto auto auto;
+    gap: 30px;
+    background-color: none;
+    padding: 45px 0px 100px 0;
+}
+
+.grid-card {
+    background-color: white;
+    text-align: center;
+    display: flex;
+    width: 328px;
+    padding: 35px 40px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 15px;
+    border: 1px solid #E8E8E8;
+}
+
+
+.card__title {
+    color: #252B42;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    text-align: center;
```

Comment:
as you already set the `text-align` property to `grid-card` you don't need to overwrite it here as well.

By the way, you even don't need to align it center even in `grid-card` as by design it should not be centered.
It's enough to just set it justify on your `card__text` (which you already did)

Reply:
Took into account. Thank you!